### ISSUE TITLE
[Feature Request] Dedicated Bookmarks Search

### ISSUE THREAD

**Issue by Glint-Eye** [tadashi-aikawa/obsidian-another-quick-switcher#208]
I really love the quick navigation that is possible by consecutively accessing outlinks and backlinks directly from the AQS prompt. But I find it always a bit difficult to find a meaningful starting point to traverse my notes. 

I think a great starting point could be utilizing **Bookmarks as a dedicated search target like "file, link, backlink, 2-hop"** in AQS. Right now, the "Star" sort priority is a bit cumbersome for this, as it doesn't filter out all the other notes and most importantly (for me at least), it doesn't **sort the starred/bookmarked files by order of appearance in the bookmark pane.** To access my bookmarked files, i still use quick switcher++, which works too, but i always have to switch to AQS afterwards to traverse through links, open pdfs etc.
If I quickly want to go through several files and then realize "oh it was actually that other file connected to my other bookmark" i always have to repeat that process of switchting between QuickSwitcher++ and AQS.

A translation of the new grouping/folder structure for bookmarks would be a nice extra too. QuickSwitcher++ solved this by Naming the entry for a grouped bookmark in a Path format like "Group A/Bookmark A". I think that is a good comprimise.

It would be nice to represent all the possible entries of the Bookmarks Pane, like bookmarks to Heading, Search Queries, Folders  etc.
**Basically a mirroring of the bookmarks pane through AQS**

Cheers and always a big thank you for the great work :)

**Comment by tadashi-aikawa** [comment]
Thank you for your FR, @Glint-Eye  :)

Personally, I don't need it very much because I don't use the bookmark feature. However, if other people need it, I'll consider it not as the "target" but as a dedicated bookmark search 👍
